src/components/Nav.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Nav = () => (
+  <nav className="nav-bar">
+    <div className="nav-items">
+      <h2>Bookstore CMS</h2>
+      <ul className="nav-links">
+        <li><Link to="/">Books</Link></li>
+        <li><Link to="catagory">Catagories</Link></li>
```

Comment:
- [ ] [OPTIONAL] If you're going to use categories as your route as suggested, pls kindly change this to `/categories` to prevent `No routes matched location "category"`

```suggestion
        <li><Link to="/categories">Catagories</Link></li>
```